website/backend/highlights/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django_extensions.db.models import TimeStampedModel
+from cloudinary.models import CloudinaryField
+from author.decorators import with_author
+
+# Create your models here.
+@with_author
+class Tag(TimeStampedModel):
+    name = models.CharField(max_length=20, null=False, blank=False)
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
When inheriting from the `BaseModel` or `TimeStampedModel`, the created_at and updated_at fields are auto added for you.